bindata/oauth-apiserver/deploy.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         apiserver: "true"
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
-        openshift.io/required-scc: privileged
+        openshift.io/required-scc: node-exporter
```

Comment:
this looks weird/wrong.  Why are we depending on an SCC from the monitoring operator?